student-work/2022/Jonathan-Kemp/api/GET.md

```diff
@@ -0,0 +1,77 @@
+# API for the General Putnam Motel Diner
+
+
+#### ExampleErrorType
+
+| Status          | Type     | Code Description                                      |
+|----------------|----------|--------------------------------------------------|
+| 200 - OK    | string  | Order is correct. |
+| 400 - Bad Request | string | Meal ordered is incorrect |
+
+## GET Retrieve the currently available meal 
+Retrieves the meal that is available at the current time. 
+## GET /meal 
+Arguments (Query Parameters) 
+There are no arguments. 
+
+#### Header parameters
```

Comment:
In the description please make it sound natural and group the choices together.
For example:
| PattySize |  string  |   Required |     Selects the type of patty for the hamburger.           |
                                                         Choose from: smallPatty, mediumPatty, largePatty |


(it would be helpful if we knew the weight of each one but not required).